New package: FittedItemBankPlots v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: FittedItemBankPlots
- Repository: https://github.com/JuliaPsychometricsBazaar/FittedItemBankPlots.jl
- Created by: @frankier
- Version: v0.1.0
- Commit: 78bb70a06641615cab46068b50aaac36b2d97911
- Reviewed by: @frankier
- Reference: https://github.com/JuliaPsychometricsBazaar/FittedItemBankPlots.jl/commit/78bb70a06641615cab46068b50aaac36b2d97911#commitcomment-133561784
- Description: Plots for FittedItemBanks.jl
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d339ccf0bede412dafa6e3569c134833f1cf13ac9053cf6888bd3ab6dd1c70d4a47864113020ad1b9f17958ae9f4de8f753c7d4064b2d5125c0338a439897504a604f67b5a8feeaf438d651e11f584fdf1de738da4ccbd209fb9e4f1b73b35167fff8c3d199a987e4cc8cc5c5fcdaec6a3747d5dd89a6e17ebe1e4bf45f0813db0f81f61db4a0a40e89e4c239fe66b1569b198570152755883b416dd15919bf6a9d698f99296dd496c1e450ea198072455fbd2342a9ffd2939601eecdb38aeba -->

goerz commented:
Please make sure to add some documentation before registering. At a minimum, this could be an explanation of the package's purpose and a basic usage example in the README.

frankier commented:
[noblock]

It's a good point. I have now provided API docs and examples will be forthcoming in the form of notebook using the package together with other stuff.

JuliaRegistrator committed:
New package: FittedItemBankPlots v0.1.0

UUID: 699f6e07-ca8e-462a-a02e-f874d7885aec
Repo: https://github.com/JuliaPsychometricsBazaar/FittedItemBankPlots.jl.git
Tree: fa34cb6d9d1c0efb29c821b7c4bda4127c948f06

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

frankier commented:
@JuliaRegistrator cancel

goerz commented:
This PR will vibe automatically closed after 30 days of inactivity
